roles/edpm_ovs_dpdk/tasks/configure.yml

```diff
@@ -196,9 +196,30 @@
       ansible.builtin.set_fact:
         edpm_ovs_other_config: "{{ edpm_ovs_other_config | combine(pmd_auto_lb_rebal_interval) }}"
 
+- name: OVN bridge datapath type config
+  when: edpm_ovn_datapath_type|string
+  block:
+    - name: Apply PMD cores config
+      ansible.builtin.set_fact:
+        ovn_bridge_datapath_type:
```

Comment:
using facts is not free for deployment time, could you please instead call combine directly with edpm_ovn_datapath_type ?

Reply:
@bogdando This section is created to cover any new OvS DPDK external ids config later similar like ovs dpdkother configs but i have added only one parameter right now.